addons/account/models/account_account.py

```diff
@@ -502,13 +513,33 @@ def _get_closest_parent_account(self, accounts_to_process, field_name, default_v
     @api.depends('account_type')
     def _compute_include_initial_balance(self):
         for account in self:
-            account.include_initial_balance = account.account_type not in ('income', 'income_other', 'expense', 'expense_depreciation', 'expense_direct_cost', 'off_balance')
+            account.include_initial_balance = account.account_type not in EXCLUDE_INITIAL_BALANCE_TYPES
+
+    def _search_include_initial_balance(self, operator, value):
+        if operator not in ['=', '!='] or not isinstance(value, bool):
+            raise UserError(_('Operation not supported'))
+        if operator != '=':
+            value = not value
+        return [('account_type', 'not in' if value else 'in', EXCLUDE_INITIAL_BALANCE_TYPES)]
+
+    def _get_internal_group(self, account_type):
+        return 'off_balance' if account_type == 'off_balance' else account_type.split('_', maxsplit=1)[0]
```

Comment:
what do you think about changing the `off_balance` selection option to `off`?
it would simplify the `_field_to_sql` too

Reply:
I thought about it too, wasn't sure if it was worth; but I'll do it :+1: